examples/watch/watch.rb

```diff
@@ -20,6 +20,6 @@
 
 watch = Kubernetes::Watch.new(client)
 
-watch.connect('/api/v1/namespaces') do |obj|
+watch.connect('/api/v1/namespaces', nil) do |obj|
```

Comment:
I think it makes sense to annotate these examples if we have more than one param and especially if the value is nil

Reply:
this is fixed by the comment above....